angular/projects/spark-angular/src/lib/components/sprk-autocomplete/sprk-autocomplete.stories.ts

```diff
@@ -0,0 +1,599 @@
+import { storyWrapper } from '../../../../../../.storybook/helpers/storyWrapper';
+import { SprkAutocompleteModule } from './sprk-autocomplete.module';
+import { SprkAutocompleteComponent } from './sprk-autocomplete.component';
+import { SprkInputModule } from '../../directives/inputs/sprk-input/sprk-input.module';
+import { SprkInputContainerModule } from '../inputs/sprk-input-container/sprk-input-container.module';
+import { SprkIconModule } from '../sprk-icon/sprk-icon.module';
+import { SprkIconComponent } from '../sprk-icon/sprk-icon.component';
+import { SprkLabelModule } from '../../directives/inputs/sprk-label/sprk-label.module';
+import { SprkLabelDirective } from '../../directives/inputs/sprk-label/sprk-label.directive';
+import { SprkIconInputContainerModule } from '../inputs/sprk-icon-input-container/sprk-icon-input-container.module';
+import { SprkInputContainerComponent } from '../inputs/sprk-input-container/sprk-input-container.component';
+import { SprkFieldErrorModule } from '../../directives/inputs/sprk-field-error/sprk-field-error.module';
+import { SprkInputDirective } from '../../directives/inputs/sprk-input/sprk-input.directive';
+import { SprkFieldErrorDirective } from '../../directives/inputs/sprk-field-error/sprk-field-error.directive';
+import { markdownDocumentationLinkBuilder } from '../../../../../../../storybook-utilities/markdownDocumentationLinkBuilder';
+import { SprkAutocompleteResultsDirective } from './sprk-autocomplete-results/sprk-autocomplete-results.directive';
+import { SprkAutocompleteResultsModule } from './sprk-autocomplete-results/sprk-autocomplete-results.module';
+import { SprkAutocompleteResultDirective } from './sprk-autocomplete-result/sprk-autocomplete-result.directive';
+import { SprkAutocompleteResultModule } from './sprk-autocomplete-result/sprk-autocomplete-result.module';
+
+export default {
+  title: 'Components/Autocomplete',
+  component: SprkAutocompleteComponent,
+  subcomponents: {
+    SprkInputContainerComponent,
+    SprkLabelDirective,
+    SprkIconComponent,
+    SprkInputDirective,
+    SprkAutocompleteResultsDirective,
+    SprkAutocompleteResultDirective,
+    SprkFieldErrorDirective,
+  },
+  decorators: [
+    storyWrapper(
+      (storyContent) => `<div class="sprk-o-Box">${storyContent}<div>`,
+    ),
+  ],
+  parameters: {
+    info: `
+${markdownDocumentationLinkBuilder('autocomplete')}
+- Live demos of the Autocomplete are available on our
+[demo site](https://spark-testangular.netlify.app/autocomplete)
+([View Source](https://github.com/sparkdesignsystem/spark-starter-angular/tree/kitchen-sink/src/app/spark-docs/autocomplete-docs)).
```

Comment:
These URLs are currently dead links. These pages will be added as part of 2467748.